staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -75,7 +75,7 @@ func init() {
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
 	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
 	AdvancedAuditing:        {Default: true, PreRelease: utilfeature.Beta},
-	APIResponseCompression:  {Default: false, PreRelease: utilfeature.Alpha},
+	APIResponseCompression:  {Default: true, PreRelease: utilfeature.Beta},
```

Comment:
also need to change in `pkg/features/kube_features.go`

Reply:
thanks for catching that